io/net/sriov_device_test.py

```diff
@@ -169,6 +251,32 @@ def get_logical_port_id(self, mac):
         logical_port_id = output.stdout_text.split(',')[6].split('=')[-1]
         return logical_port_id
 
+    def get_hnv_bond(self):
+        """
+        Get the newly created hnv bond interface name
+        """
+        output = process.system_output('cat /sys/class/net/bonding_masters',
+                                       ignore_status=True, shell=True,
+                                       sudo=True).decode("utf-8").split()
```

Comment:
`genio.read_file()`

Reply:
Yeap, I replaced with genio way of reading file